test/integration/fake_upstream.h

```diff
@@ -569,7 +569,8 @@ class FakeUpstream : Logger::Loggable<Logger::Id::testing>,
                      std::chrono::milliseconds timeout = TestUtility::DefaultTimeout);
 
   // Send a UDP datagram on the fake upstream thread.
-  void sendUdpDatagram(const std::string& buffer, const Network::Address::Instance& peer);
+  void sendUdpDatagram(const std::string& buffer,
```

Comment:
Why making this change?

Reply:
This is the fix for https://github.com/envoyproxy/envoy/issues/9018